src/userLog.js

```diff
@@ -1,7 +1,49 @@
+"use strict";
+
 var fs = require("fs");
 var fsextra = require("fs-extra");
 var moment = require("moment");
 var Helper = require("./helper");
+const Msg = require("./models/msg");
+
+module.exports.read = function(user, network, chan) {
+	var data = fs.readFileSync(
```

Comment:
Agreed. So far this has only been a POC so I did not care for early optimization, but now that I have something that works fairly well, I'll start giving it some thought.

Thanks for the input, very useful!